src/libs/tiny_ttf/lv_tiny_ttf.h

```diff
@@ -29,26 +29,60 @@ extern "C" {
  * GLOBAL PROTOTYPES
  **********************/
 
-#if LV_TINY_TTF_FILE_SUPPORT !=0
-/* create a font from the specified file or path with the specified line height.*/
+#if LV_TINY_TTF_FILE_SUPPORT != 0
+/**
+ * create a font from the specified file or path with the specified line height.
```

Comment:
Start function title with capital letter.
```suggestion
 * Create ....
```